### PR TITLE
Adding ZstarTophiATo4tau cards

### PR DESCRIPTION
**Pull request by gputtley**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi100A140To4Tau/ZstarTophi100A140To4Tau_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi100A140To4Tau/ZstarTophi100A140To4Tau_proc_card.dat:15]: those lines can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi300A160To4Tau/ZstarTophi300A160To4Tau_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi300A160To4Tau/ZstarTophi300A160To4Tau_proc_card.dat:15]: these lines can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi100A140To4Tau/ZstarTophi100A140To4Tau_proc_card.dat (hunk 1 above)
Comment (gputtley) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi100A140To4Tau/ZstarTophi100A140To4Tau_proc_card.dat]: These have been removed

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi300A160To4Tau/ZstarTophi300A160To4Tau_proc_card.dat (hunk 2 above)
Comment (gputtley) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZstarTophiATo4tau/ZstarTophi300A160To4Tau/ZstarTophi300A160To4Tau_proc_card.dat]: These have been removed

**Comment by gputtley** [comment]
Cards have been moved to 2017/13TeV

**Comment by menglu21** [comment]
thanks for the update. @Saptaparna @mlizzo @Dominic-Stafford please have a check

**Pull request merged**